Backend/src/db/setters.py

```diff
@@ -26,3 +27,39 @@ def add_multiple_nodes_to(
     nodes_to_insert = [one_node.dict() for one_node in nodes]
     with ConnectionClass() as (_, db):
         db[collection].insert_many(nodes_to_insert)
+
+
+@validate_endpoint_args
+def add_multiple_nodes_to_id(
+    nodes: list[Node], collection: str, document_id: str, ConnectionClass=MongoConnection
+) -> None:
+    """
+    Bulk function to adding multiple nodes at once to a specific id.
```

Comment:
Rewrite suggestion:  Bulk function for adding multiple nodes at once to a specific id.